### PR TITLE
[Manual] Release v2.38.0

### PR DESCRIPTION
**Pull request by tbekas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+## [v2.38.0] 2022-09-03
+
+### Changed
+
+- Whitelisting update
+- DTM rewards update
+- Soft staking nodes for September
+
 ## [v2.37.0] 2022-08-03
+
```
Comment (marcinwadon) [inline CHANGELOG.md:16]: R

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,21 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [v2.38.0] 2022-09-03
+
```
Comment (marcinwadon) [inline CHANGELOG.md:8]: R

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+### Changed
+
+- Whitelisting update
+- DTM rewards update
+- Soft staking nodes for September
+
 ## [v2.37.0] 2022-08-03
+
 ### Changed
+
```
Comment (marcinwadon) [inline CHANGELOG.md:18]: R

**Review by marcinwadon — DISMISSED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (tbekas) [reply CHANGELOG.md]: Thanks for catching it. Autoformatter in Intellij :/

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**